### PR TITLE
Add group abbreviation field. Displaying and editing works

### PR DESCRIPTION
**Pull request by BSteephenson**
I
- made a group abbreviation field (called it abb for short) using the specs from issue #18 
- edited the forms in modals.jade to add this field
- made it so in the side bar, if there's an abbreviation, show that instead of the group name
- editing the abbreviation works

### CODE REVIEW

File: tuftslife/views/events/modals.jade [hunk 1]
```diff
@@ -12,6 +12,10 @@ div.modal.fade.tl-addGroupModal
 						div.col-sm-9
 							input.form-control.tl-addGroup-name
 					div.form-group
+						label.control-label.col-sm-3 Abbreviation (not required)
```
Comment (zfrenchee) [inline tuftslife/views/events/modals.jade]: "(not required)" bugs me a little. Can we do better? How about stars next to required fields? (would that make sense to add to this PR?)

File: tuftslife/assets/js/EventsPageView.coffee [hunk 2]
```diff
@@ -61,10 +61,13 @@ class EventsPageView extends Backbone.View
...
 					appendDemarcation(startTime.format('dddd, MMMM Do YYYY'))
 
+				# Use the group abbreviation if possible, else use the name
+				groupName = event.get('owner').abb || event.get('owner').name
+
 				$('.tl-events').append """
 					<div class='tl-event' id='#{event.get('id')}'>
 						<div class='tl-event-time'>#{startTime.format('h:mm a')}</div>
-						<span class='tl-event-title'>#{event.get('name')}</span>
+						<span class='tl-event-title'>#{groupName}</span>
```
Comment (zfrenchee) [inline tuftslife/assets/js/EventsPageView.coffee]: <span class='tl-event-title'>#{event.get('owner').abb || event.get('owner').name}</span>
in my opinion.

**Comment by zfrenchee** [comment]
Excellent stuff. Would you mind attaching a screenshot just so we can take a look?

File: tuftslife/views/events/modals.jade (hunk 1 above)
Comment (BSteephenson) [inline tuftslife/views/events/modals.jade]: Should we also explicitly say that the ones with stars are required?

**Comment by BSteephenson** [comment]
The most rivetting 53 seconds of your life
https://vid.me/25CW

**Comment by BSteephenson** [comment]
Just realized i was changing the event name, not the group name =P
The stars look ugly. I just put a "(optional)" placeholder in the field. Make it perfect later.
To merge.

**Comment by zfrenchee** [comment]
Although the video was indeed riveting.... exactly

**Pull request merged**